examples/mutate/abstract functionality blueprint/mutate.py

```diff
@@ -24,7 +24,7 @@ def data_3() -> pd.DataFrame:
 
 # data1 and data2
 @mutate(data_1, data_2)
-def _filter(some_data: pd.DataFrame) -> pd.DataFrame:
```

Comment:
so the way this works, is that hamilton's find functions will create a node for this function, but with the decorator we inject `ignore` so that when we come to resolve nodes, they resolve to None.

Is that right?

Reply:
@skrawcz nope, this is a `NodeResolver` so it will short-circuit, and not create a node for the function. Effectively Hamilton won't see a function when it attempts to parsse (and thus it's not bound to the same rules).

Reply:
I also checked and the following will not raise an error of missing type hints, so this allows you to have non-hamilton functions without the "_" pre-fix:

```python
@ignore
def foo(a,b,c):
    return a+b+c
```